types/craftos/craftos.d.ts

```diff
@@ -140,680 +150,775 @@ declare var _G: LuaTable;
-    function setFocus(n: number): void;
-    function getTitle(n: number): string | null;
-    function setTitle(n: number, title: string): void;
-    function getCurrent(): number;
-    function launch(env: LuaTable, path: string, ...args: string[]): number;
-    function getCount(): number;
+  function getFocus(): number;
+  function setFocus(n: number): void;
+  function getTitle(n: number): string | null;
+  function setTitle(n: number, title: string): void;
+  function getCurrent(): number;
+  function launch(env: LuaTable, path: string, ...args: string[]): number;
+  function getCount(): number;
 }
 declare type LuaDate = {
-    year: number;
-    month: number;
-    day: number;
-    hour: number;
-    min: number;
-    sec: number;
-    wday: number;
-    yday: number;
-    isdst: boolean;
-}
+  year: number;
+  month: number;
+  day: number;
+  hour: number;
+  min: number;
+  sec: number;
+  wday: number;
+  yday: number;
+  isdst: boolean;
+};
 
 declare namespace os {
-    function version(): string;
-    function getComputerID(): number;
-    function computerID(): number;
-    function getComputerLabel(): string | null;
-    function computerLabel(): string | null;
-    function setComputerLabel(label?: string | null): void;
-    function run(env: LuaTable, path: string, ...args: string[]): boolean;
-    function queueEvent(type: string, ...param: any[]): void;
-    function clock(): number;
-    function startTimer(timeout: number): number;
-    function cancelTimer(id: number): void;
-    function time(mode?: string | null): number;
-    function sleep(timeout: number): void;
-    function day(mode?: string | null): number;
-    function setAlarm(time: number): number;
-    function cancelAlarm(id: number): void;
-    function shutdown(): void;
-    function reboot(): void;
-    function epoch(mode?: string | null): number;
-    function date(format?: string | null, time?: number | null): string | LuaDate;
+  function loadAPI(path: string): boolean;
+  function unloadAPI(name: string): void;
+  function version(): string;
+  function getComputerID(): number;
+  function computerID(): number;
+  function getComputerLabel(): string | null;
+  function computerLabel(): string | null;
+  function setComputerLabel(label?: string | null): void;
+  function run(env: LuaTable, path: string, ...args: string[]): boolean;
+  function queueEvent(type: string, ...param: any[]): void;
+  function clock(): number;
+  function startTimer(timeout: number): number;
+  function cancelTimer(id: number): void;
+  function time(mode?: string | null): number;
+  function sleep(timeout: number): void;
+  function day(mode?: string | null): number;
+  function setAlarm(time: number): number;
+  function cancelAlarm(id: number): void;
+  function shutdown(): void;
+  function reboot(): void;
+  function epoch(mode?: string | null): number;
+  function date(format?: string | null, time?: number | null): string | LuaDate;
+}
+declare namespace turtle {
```

Comment:
As requested above, please make sure the new APIs match the code style of the original code.